GetComponents

```diff
@@ -385,7 +384,7 @@ sub parse_list {
     }
     # if $ROOT is undefined, it will be set to the current directory
     if   ( defined( $DEFINITIONS{ROOT} ) ) { $ROOT = $DEFINITIONS{ROOT} }
-    else                                   { $ROOT = $orig_dir }
+    else                                   { $ROOT = cwd() }
```

Comment:
Right. So I should use "." instead of cwd(). I just tested this and indeed with cwd() I get errors if I remove the ROOT define from the ET thornlist but using "." works fine. I amended the pull request.